### PR TITLE
Error on bad type

### PR DESCRIPTION
**Pull request by hozfia**
i added simple function to check for bad type files and display message of the file(s) name witch are not supported.  

the supported type are listed in array , i don't know if this proper way because it mean that you need to list all the types you are supporting , i listed a few of them witch ware listed in other parts of the code 

here screenshot 
memb

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1391,9 +1391,49 @@ function exportVideo(blob) {
   vid.currentTime = Number.MAX_SAFE_INTEGER;
 }
 
+function uploadSupportedType(files) {
+  let supportedType = [
```
Comment (bwasti) [inline script.js]: Let's just use the `ext_map` on line 1192. We can make that global and then use `if (!(extension in ext_map)) {`.

File: script.js [hunk 2]
```diff
@@ -1391,9 +1391,49 @@ function exportVideo(blob) {
...
+  for (let file of files) {
+    let extension = file.name.split('.').pop();
+    if (!supportedType.includes(extension)) {
+      badUserExtensions.push(file)
+    }
+  }
+
+  (badUserExtensions.length > 0)?
+  alert('the file(s) you uploaded are not supported \n' 
+  + badUserExtensions.map((ext)=>"- "+ext.name).join('\n')):"";
```
Comment (bwasti) [inline script.js]:
```suggestion
  if (badUserExtensions.length > 0) {
    const badFiles = badUserExtensions.map((ext)=>"- "+ext.name).join('\n');
    alert('the file(s) you uploaded are not supported \n' + badFiles);
  }
```

**Review by bwasti — CHANGES_REQUESTED** [review]
Looks great!  I appreciate the contribution.

The only main change I'd like to see is to use the `ext_map` instead of `supportedType` (the other types are for exporting and it's fine to ignore them).

If you'd like to remove the `alert`, there's a `popup` function.

**Comment by hozfia** [comment]
i added those changes : 
make ext_map global variable , then use popup instead of using browser alert  
here screenshot
Screenshot from 2021-02-04 11-57-27

**Review by bwasti — APPROVED** [review]
Awesome!

**Pull request merged**